contrib/netperf/collectd-td.conf

```diff
@@ -27,9 +27,9 @@ WriteQueueLimitLow 5000
 	ReportByConnections true
 	ReportByPorts false
 
-	TcpInfoField "rtt  GAUGE  uint32:68  rcvd_ack"
-	TcpInfoField "total_retrans  GAUGE uint32:100"
-	TcpInfoField "rttvar  GAUGE uint32:72   rcvd_ack"
+	TcpInfoField "rtt		GAUGE   uint32:68   rcvd_ack"
+	TcpInfoField "delta_retrans	COUNTER uint32:100"
```

Comment:
This is of course correct! I'm hesitant to introduce another level of configuration syntax. And I'm willing to override the collectd definition of COUNTER, mostly because it's hardly ever used in upstream collectd.
So I'm keeping it as-is, and if we ever make the change (maybe due to merging upstream) I'll add a fix on our backend to handle both formats.